build.gradle

```diff
@@ -366,7 +366,8 @@ project(':ambry-cloud') {
             project(':ambry-store'),
             project(':ambry-commons'),
             project(':ambry-replication'),
-            project(':ambry-router')
+            project(':ambry-router'),
```

Comment:
What does ambry-cloud use the router dependency for?

Reply:
fixed.

Reply:
only test required it. fixed.